src/ui-test/utils/testUtils.ts

```diff
@@ -380,3 +380,21 @@ export async function getCamelCatalogVersion(): Promise<string> {
 	await new EditorView().closeEditor('Settings');
 	return value;
 }
+
+/**
+ * Creates '.properties' file with required name.
+ * 
+ * @param driver WebDriver. 
+ * @param name Required name for new properties file. 
+ */
+export async function createPropertiesFile(driver: WebDriver, name: string): Promise<void> {
```

Comment:
seems like general `createFile` method, it is not limited to spawn only `.properties` files